packages/java/endpoint/src/main/java/dev/hilla/crud/PropertyStringFilterSpecification.java

```diff
@@ -90,4 +144,15 @@ private boolean isBoolean(Class<?> javaType) {
         return javaType == boolean.class || javaType == Boolean.class;
     }
 
+    private boolean isLocalDate(Class<?> javaType) {
+        return javaType == java.time.LocalDate.class;
+    }
+
+    private boolean isLocalTime(Class<?> javaType) {
+        return javaType == java.time.LocalTime.class;
+    }
+
+    private boolean isLocalDateTime(Class<?> javaType) {
+        return javaType == java.time.LocalDateTime.class;
+    }
```

Comment:
No support for other types that represent date and time? 
* java.sql.Date (supporting old projects that need modernization in UI)
* ZonedDateTime
* Timestamp (in many datebases this is favoured over the DATE and DATETIME db types because of being more performant in indexing, searching, storage, etc.)

Reply:
If these are needed, separate issues should be created

Reply:
https://github.com/vaadin/hilla/issues/1464